core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -873,4 +875,31 @@ static function (array $allowedProperties): array {
             ->end()
         ;
     }
+
+    private function addAltchaNode(): NodeDefinition
+    {
+        return (new TreeBuilder('altcha'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->enumNode('algorithm')
+                    ->info('Choose an algorithm which is required for generating the challenges. Select between "SHA-256", "SHA-384" or "SHA-512".')
+                    ->values([...array_column(AlgorithmConfig::cases(), 'value')])
```

Comment:
I'm not sure but imho Symfony has native enum support so you can just go with `Algorithm::cases()` but you'll have to double check.